### ISSUE TITLE
pkill cow无效

### ISSUE THREAD

**Issue by nickyfoto** [cyfdecyf/cow#350]
新版本pkill 后会自动重启，如何彻底结束cow进程？
OS X 10.10.4

**Comment by fzerorubigd** [comment]
restart should happen with -USR1 signal (pkill -USR1 cow) normal pkill send a TERM, and with the current dev snapshot, there is no problem for me. 

-- I use google translator to understand you problem, i hope I understand it well :)

**Comment by nickyfoto** [comment]
Thanks fzerorubigd. I want to know how to kill cow permanently. with `pkill -USR1 cow`?

**Comment by fzerorubigd** [comment]
No. to kill it permanently, just pkill cow would be enough, -USR1 is used
to reload it.

On Wed, Jul 8, 2015 at 6:00 PM, Huang Qiang <notifications@github.com>
wrote:

> Thanks fzerorubigd. I want to know how to kill cow permanently. with pkill
> -USR1 cow?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyfdecyf/cow/issues/350#issuecomment-119578721>.
>



-- 
-----------------------------
http://fzero.rubi.gd

**Comment by tzhao11** [comment]
@nickyfoto 
默认情况下 cow 在 OS X 上是作为 LaunchAgent 运行的，由 launchd 守护。要彻底结束 cow 请运行： `launchctl unload ~/Library/LaunchAgents/info.chenyufei.cow.plist`

**Comment by nickelchen** [comment]
@tzhao11 works like a charm! thank you
